services/app-api/src/authn/thirdPartyAuthn.ts

```diff
@@ -0,0 +1,26 @@
+import type { Result } from 'neverthrow'
+import { ok, err } from 'neverthrow'
+import type { UserType } from '../domain-models/index'
+import type { Store } from '../postgres'
+import { lookupUserAurora } from './cognitoAuthn'
+
+export async function userFromThirdPartyAuthorizer(
```

Comment:
you could probably have the type def of this function be explicitly the userFromAuthProvider type

Reply:
I didn't use that here because `authProvider` is required with userFromAuthProvider type and that wouldn't be used in this function.